cogs/general.py

```diff
@@ -59,9 +63,45 @@ def __init__(self, bot):
     async def twitter(self):
         await s.destructmsg('https://twitter.com/idiotechgaming', 20, self.bot)
 
+    @commands.command(description=desc.fb, brief=desc.fb)
+    async def facebook(self):
+        with aiohttp.ClientSession() as session:
+            async with session.get('https://graph.facebook.com/v2.6/idiotechgaming/posts'
+                                   '?access_token={}'.format(t.fb_key)) as resp:
+                data = await resp.json()
+
+                page = "https://www.facebook.com/idiotechgaming/"  # FaceBook Page link
+
+                msg1 = data["data"][0]["message"]
+                y,m,d, = date_split(data["data"][0]["created_time"])  # y = year, m = month, d = day
+
+                time = "**Posted:** {}{} of {}, {}.\n\n".format(d, get_date_suf(d), calendar.month_name[int(m)], y)
+                title = "**Latest Facebook Post**\n"
+
+                await s.destructmsg(title+time+"```"+msg1+"```\n"+page, 30, self.bot)
```

Comment:
I prefer having a message in multiline string ("""), with one big format function. That way you glance at the code and know what the message is going to look like